<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditLevelDialog</class>
 <widget class="QDialog" name="EditLevelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Level</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="level_name_label">
       <property name="text">
        <string>Level name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="script_name_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="script_name_label">
       <property name="text">
        <string>Script file name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="folder_name_label">
       <property name="text">
        <string>Script folder name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="extension_label">
       <property name="text">
        <string>.term.txt</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="level_name_edit"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="folder_name_edit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialog_button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
